webpack.config.js

```diff
@@ -74,29 +74,6 @@ module.exports = {
           }
         ]
       },
-      {
-        test: /\.(woff2?|ttf|otf|eot)$/,
-        loader: 'file-loader',
-        options: {
-          name: '[name].[ext]',
-          publicPath: './assets/fonts',
-          outputPath: '../dist/assets/fonts'
-        }
-      },
-      // Apply plugins to image assets.
-      {
-        test: /\.(png|svg|jpg|gif)$/i,
-        use: [
-          {
-            loader: 'file-loader',
-            options: {
-              name: '[name].[ext]',
-              publicPath: './assets/img',
-              outputPath: '../dist/assets/img'
-            }
```

Comment:
@sherakama I removed these from webpack.config since we probably won't have fonts/images in this repo now. Is that ok? Could we also remove the asset/ image and font folders?

Reply:
Move it to `/dev/webpack.dev.config.js` just in case we need it there.